nrf_dm/doc/nrf_dm_usage.rst

```diff
@@ -24,8 +24,7 @@ You can use the Distance Measurement library as follows:
 
    You can meet this synchronization requirement by starting the reflector earlier and increasing the timeout value, at the cost of higher power consumption.
 
-#. Call the :c:func:`nrf_dm_calc` function to perform the calculation.
-   The calculation must be performed between the measurements to obtain a valid result.
+#. Load the raw data using the :c:func:`nrf_dm_populate_report` function.
 
-#. Call the :c:func:`nrf_dm_report_get` function to get the results.
-   It returns a pointer to the :c:struct:`nrf_dm_report_t` structure which contains the :c:struct:`nrf_dm_dist_estimates_t` structure with an estimation of the distance.
+#. Call the :c:func:`nrf_dm_calc` function or the :c:func:`nrf_dm_high_precision_calc` function to perform the calculation.
+   On the nRF53 Series, this step should preferably be done on the application core for maximum performance.
```

Comment:
Maybe we should specify that performance you are talking about is execution time (and not anything to do with the accuracy of the estimates)